File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int idx = s.length() - 1, ans = 0;
        while(idx >= 0 && s.charAt(idx) == ' ') idx--;
        while(idx >= 0 && s.charAt(idx) != ' ') {
            idx--;
            ans++;
        }

        return ans;
    }
}

File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int idx = s.size() - 1, ans = 0;
        while(idx >= 0 && s[idx] == ' ') idx--;
        while(idx >= 0 && s[idx] != ' ') {
            ans++;
            idx--;
        }

        return ans;
    }
};